[
    {
        "instance_id": "Capsa_CVE-2022-26049",
        "repo": "diffplug/goomph",
        "base_commit": "fe2083196f0d0a75885aea402baaa972254173d5",
        "patch_commit": "25f04f67ba62d9a14104bee13a0a0f2517afb8c8",
        "vuln_file": "src/main/java/com/diffplug/gradle/ZipMisc.java",
        "vuln_lines": [177, 190],
        "language": "java",
        "vuln_source": "CVE-2022-26049",
        "cwe_id": "cwe-22",
        "vuln_type": "Path Traversal",
        "severity": "high",
        "image": "d0cker312/cve-2022-26049:v2",
        "image_name": "sjtu_03",
        "image_inner_path": "/app/goomph/",
        "image_run_cmd": "tail -f /dev/null",
        "image_status_check_cmd": "./image_status_check.sh",
        "test_case_cmd": "./test_case.sh",
        "poc_cmd": "./poc.sh",
        "help_commit_urls": [
        "https://github.com/diffplug/goomph/commit/25f04f67ba62d9a14104bee13a0a0f2517afb8c8"
        ],
        "help_poc": {
        "pocCode": [
            {
            "date": "",
            "desc": "",
            "source": "nvd",
            "url": "https://security.snyk.io/vuln/SNYK-JAVA-COMDIFFPLUGGRADLE-2981040"
            }
        ],
        "pocUrl": [
            "https://security.snyk.io/vuln/SNYK-JAVA-COMDIFFPLUGGRADLE-2981040"
        ]
        }
    },
    {
        "instance_id": "Capsa_CVE-2022-23457",
        "repo": "ESAPI/esapi-java-legacy",
        "base_commit": "d34beae9860d73df4af38f1c495c669b06ed8694",
        "patch_commit": "a0d67b75593878b1b6e39e2acc1773b3effedb2a",
        "vuln_file": "src/main/java/org/owasp/esapi/reference/DefaultValidator.java",
        "vuln_lines": [448, 482],
        "language": "java",
        "vuln_source": "CVE-2022-23457",
        "cwe_id": "cwe-22",
        "vuln_type": "Path Traversal",
        "severity": "high",
        "image": "d0cker312/cve-2022-23457",
        "image_name": "sjtu_05",
        "image_inner_path": "/src/esapi-java-legacy",
        "image_run_cmd": "tail -f /dev/null",
        "image_status_check_cmd": "bash -c ./image_status.sh",
        "test_case_cmd": "./test_case.sh",
        "poc_cmd": "./poc.sh",
        "help_commit_urls": [
        "https://github.com/ESAPI/esapi-java-legacy/commit/a0d67b75593878b1b6e39e2acc1773b3effedb2a"
        ],
        "help_poc": {
        "pocCode": [
            {
            "date": "",
            "desc": "",
            "source": "nvd",
            "url": "https://securitylab.github.com/advisories/GHSL-2022-008_The_OWASP_Enterprise_Security_API/"
            }
        ],
        "pocUrl": [
            "https://securitylab.github.com/advisories/GHSL-2022-008_The_OWASP_Enterprise_Security_API/"
        ]
        }
    },
    {
        "instance_id": "Capsa_CVE-2022-21675",
        "repo": "Konloch/bytecode-viewer",
        "base_commit": "c968e94b2c93da434a4ecfac6d08eda162d615d0^",
        "patch_commit": "c968e94b2c93da434a4ecfac6d08eda162d615d0",
        "vuln_file": "src/main/java/the/bytecode/club/bytecodeviewer/util/ZipUtils.java",
        "vuln_lines": [46, 85],
        "language": "java",
        "vuln_source": "CVE-2022-21675",
        "cwe_id": "cwe-22",
        "vuln_type": "Path Traversal",
        "severity": "high",
        "image": "d0cker312/cve-2022-21675",
        "image_inner_path": "/src/bytecode-viewer",
        "image_run_cmd": "tail -f /dev/null",
        "image_status_check_cmd": "./image_status.sh",
        "test_case_cmd": "./test_case.sh",
        "poc_cmd": "./poc.sh"
    },
    {
        "instance_id": "Capsa_CVE-2024-1594",
        "repo": "mlflow/mlflow",
        "base_commit": "f8d51e21523238280ebcfdb378612afd7844eca8~",
        "patch_commit": "f8d51e21523238280ebcfdb378612afd7844eca8",
        "vuln_file": "mlflow/server/handlers.py",
        "vuln_lines": [589, 612, 1724, 1742],
        "language": "python",
        "vuln_source": "CVE-2024-1594",
        "cwe_id": "cwe-22",
        "vuln_type": "Path Traversal",
        "severity": "high",
        "image": "d0cker312/cve-2024-1594",
        "image_name": "sjtu_09",
        "image_inner_path": "/src/mlflow",
        "image_run_cmd": "tail -f /dev/null",
        "image_status_check_cmd": "/src/image_status.sh",
        "test_case_cmd": "/src/test_case.sh",
        "poc_cmd": "/src/poc.sh",
        "other_vuln_files": []
    },
    {
        "instance_id": "Capsa_CVE-2024-1593",
        "repo": "mlflow/mlflow",
        "base_commit": "v2.9.2",
        "patch_commit": "695d287ec824760474bfd21a6d139d0efedb9aee",
        "vuln_file": "mlflow/server/handlers.py",
        "vuln_lines": [579, 601, 1591, 1621],
        "language": "python",
        "vuln_source": "CVE-2024-1593",
        "cwe_id": "cwe-22",
        "vuln_type": "Path Traversal",
        "severity": "high",
        "image": "d0cker312/cve-2024-1593",
        "image_inner_path": "/src/mlflow",
        "image_run_cmd": "tail -f /dev/null",
        "image_status_check_cmd": "/src/image_status.sh",
        "test_case_cmd": "/src/test_case.sh",
        "poc_cmd": "/src/poc.sh",
        "other_vuln_files": []
    }
]